src/modules/Game.class.js

```diff
@@ -0,0 +1,254 @@
+'use strict';
+
+/**
+ * This class represents the game.
+ * Now it has a basic structure, that is needed for testing.
+ * Feel free to add more props and methods if needed.
+ */
+class Game {
+  static get deafultSize() {
+    return 4;
+  }
+
+  static get deafultInitialState() {
```

Comment:
There is a typo in the method name 'deafultInitialState'. It should be 'defaultInitialState'.